axlearn/common/utils.py

```diff
@@ -158,6 +159,8 @@ def visit(tree, prefix):
                 (k, visit(v, _concat(prefix=prefix, suffix=k, separator=separator)))
                 for k, v in tree.items()
             )
+        elif isinstance(tree, flax.struct.PyTreeNode):
```

Comment:
Is there a test that can be added to catch the issue?

Reply:
Added.